crates/ruma-identity-service-api/src/versions.rs

```diff
@@ -0,0 +1,44 @@
+//! `GET /_matrix/identity/versions` ([spec])
+//!
+//! [spec]: https://spec.matrix.org/v1.2/identity-service-api/#get_matrixidentityversions
+
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Get the versions of the identity service API supported by this endpoint.",
+        method: GET,
+        name: "versions",
+        // note: The MSC does not specify an unstable endpoint, nor does it offer any fallbacks,
+        // regardless, this is only supported after v1.1, so a client has to take a gamble if an
+        // identity service supports this or not.
+        //
+        // Safe to say, if the endpoint returns a 404, then it's likely only supporting < v1.1.
```

Comment:
Would there be a better place i can put this bit of info?

Reply:
I would put this in the module description or the response's docs.